bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -119,10 +119,6 @@ const currentUserEmoji = (currentUser) => (currentUser
 );
 
 export default injectIntl(withModalMounter(withTracker(({ intl, baseControls }) => {
-  if (Auth.connectionID !== Meteor.connection._lastSessionId) {
```

Comment:
@Tainan404 could you please explain briefly why this is safe to remove?

Reply:
In the current PR, we are able to keep the client working offline and on the reconnection we re-authenticate the user, so, the connection changes before user be re-authed causing the user to be kicked out from the meeting